google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java

```diff
@@ -517,7 +524,15 @@ private AppendRequestAndResponse pollInflightRequestQueue() {
     return new StreamWriter.Builder(streamName);
   }
 
-  /** A builder of {@link StreamWriterV2}s. */
+  public TableSchema getUpdatedSchema() {
```

Comment:
You need a lock over these get and set to protect concurrent access of the object.

Reply:
Done